mathview-lib/src/main/assets/www/MathTemplate.html

```diff
@@ -33,7 +33,7 @@
 </script>
 
     <script type="text/javascript"
-            src="scripts/mathjax/2.7.0/MathJax.js?config=TeX-MML-AM_HTMLorMML"></script>
+            src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
Thanks for the Pull Request!
Instead of using the latest, can u please pinpoint a particular release version? Using the latest might break existing functionality.

Reply:
Hey so I thought that the library was outdated but it wasn't the problem I used the same version(2.7.0) of mathjax in windows and the equation was showed up perfectly
![Screenshot (52)](https://user-images.githubusercontent.com/73039980/121304330-d3d13280-c919-11eb-89d5-14609c13d649.png)
but in android the same equation is showing up like this 
![Screenshot_2021-06-09-11-57-57-1](https://user-images.githubusercontent.com/73039980/121304616-362a3300-c91a-11eb-8fd2-f71fe1095e52.png)

I went through MathView.java although I wasn't able to figure out the problem but I think the problem is there